initfiles/componentfiles/configxml/environment.xsd.in

```diff
@@ -121,6 +121,17 @@
                                     <xs:attribute name="domain" type="xs:string"/>
                                     <xs:attribute name="computerType" type="xs:string"/>
                                 </xs:complexType>
+
+                                <xs:element name="cost">
+                                    <xs:complexType>
+                                        <xs:attribute name="memoryLocal" type="xs:string"/>
```

Comment:
Perhaps in context it becomes obvious, but at least for me these attribute names are not entirely self-documenting.